tests/foreman/ui_airgun/test_dashboard.py

```diff
@@ -372,3 +444,106 @@ def test_positive_content_host_subscription_status(session):
+    # create a role with necessary permissions
+    role = entities.Role().create()
+    user_permissions = {
+        'Organization': ['view_organizations'],
+        'Location': ['view_locations'],
+        None: ['access_dashboard'],
+        'Host': ['view_hosts'],
+    }
+    create_role_permissions(role, user_permissions)
+    # create a user and assign the above created role
+    entities.User(
+        default_organization=org,
+        organization=[org],
+        default_location=module_loc,
+        location=[module_loc],
+        role=[role],
+        login=user_login,
+        password=user_password
+    ).create()
+    with Session(test_name, user=user_login, password=user_password) as session:
+        assert session.dashboard.read('HostConfigurationStatus')['total_count'] == 0
+        assert len(session.dashboard.read('LatestErrata')) == 0
+        repos_collection = RepositoryCollection(
+            distro=DISTRO_RHEL7,
+            repositories=[
+                SatelliteToolsRepository(),
+                YumRepository(url=FAKE_6_YUM_REPO),
+            ]
+        )
+        repos_collection.setup_content(org.id, lce.id)
+        with VirtualMachine(distro=repos_collection.distro) as client:
+            repos_collection.setup_virtual_machine(client)
+            result = client.run('yum install -y {0}'.format(FAKE_1_CUSTOM_PACKAGE))
+            assert result.return_code == 0
+            hostname = client.hostname
+            # Check UI for values
+            assert session.host.search(hostname)[0]['Name'] == hostname
+            hosts_values = session.dashboard.read('HostConfigurationStatus')
+            assert hosts_values['total_count'] == 1
+            errata_values = session.dashboard.read('LatestErrata')['erratas']
+            assert len(errata_values) == 1
+            assert errata_values[0]['Type'] == 'security'
+            assert FAKE_2_ERRATA_ID in errata_values[0]['Errata']
+
+
+@tier2
+def test_positive_sync_overview_widget(session):
+    """Check if the Sync Overview Widget is working in the Dashboard UI
+
+    :id: 515027f5-19e8-4f83-9042-1c347a63758f
+
+    :Steps:
+
+        1. Create a product
+        2. Add a repo and sync it
+        3. Navigate to Monitor -> Dashboard
+        4. Review the Sync Overview widget for the above sync details
+
+    :expectedresults: Sync Overview widget is updated with all sync processes
+
+    :CaseLevel: Integration
+    """
+    product_name = gen_string('alpha')
+    org = entities.Organization().create()
+    create_sync_custom_repo(org.id, product_name=product_name)
+    with session:
+        session.organization.select(org_name=org.name)
+        sync_values = session.dashboard.read('SyncOverview')['syncs']
+        assert len(sync_values) == 1
+        assert sync_values[0]['Product'] == product_name
+        assert sync_values[0]['Status'] == 'Syncing Complete.'
+        assert 'minute ago' in sync_values[0]['Finished']
```

Comment:
what if 2 minutes ago :) ?

Reply:
ahh, it will change word to plural. =) okay, I will change that